coalib/bearlib/aspects/meta.py

```diff
@@ -72,3 +75,24 @@ class Sub(subcls, aspectbase, metaclass=aspectclass):
 
     def __repr__(cls):
         return '<%s %s>' % (type(cls).__name__, repr(cls.__qualname__))
+
+
+def issubaspect(aspect, subaspect):
+    """
+    This function checks whether or not ``subcls`` is a subaspect of cls
+    """
+    if not isinstance(aspect, aspectclass):
+        logging.warning(
+            '{} is not an aspectclass instance'.format(
+                type(aspect).__name__))
+        return None
+    if not isinstance(subaspect, aspectclass):
+        logging.warning(
+            '{} is not an aspectclass instance'.format(
+                type(subaspect).__name__))
+        return None
+    parent_aspect_qualname = aspect.__qualname__ + '.+'
+    if re.match(parent_aspect_qualname, subaspect.__qualname__) is not None:
```

Comment:
Why `re.match` ? Wont this be OK. `if(aspect.__qualname__ in subaspect.__qualname__):`